thriftrpc/multiservicecall/multi_service_test.go

```diff
@@ -115,6 +115,43 @@ func TestMultiServiceWithCombineServiceClient(t *testing.T) {
 	test.Assert(t, resp.Message == "servicea Echo1")
 }
 
+func TestUnknownException(t *testing.T) {
+	ip := "localhost:9907"
+	svr := GetServer(ip)
+	servicea.RegisterService(svr, new(ServiceAImpl))
+	go svr.Run()
+	defer svr.Stop()
+	time.Sleep(100 * time.Millisecond)
+
+	clientB, err := serviceb.NewClient("ServiceB",
+		client.WithHostPorts(ip),
+		client.WithTransportProtocol(transport.TTHeader),
+		client.WithMetaHandler(transmeta.ClientTTHeaderHandler))
+	test.Assert(t, err == nil, err)
+	_, err = clientB.Echo2(context.Background(), &multi_service.Request{Message: "multi_service req"})
+	test.Assert(t, err != nil)
+	test.DeepEqual(t, err.Error(), "remote or network error[remote]: unknown method Echo2")
+}
+
+func TestUnknownExceptionWithMultiService(t *testing.T) {
```

Comment:
this is to check unknown service.
I think we also need to add one more test to check unknown methon in multi-service